src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/SystemGroovyChoiceListProviderJenkinsTest.java

```diff
@@ -627,28 +645,20 @@ public void testApproval() throws Exception
         final String SCRIPT = "return java.util.Arrays.asList(new File('/').list());";
         
         FreeStyleProject p = j.createFreeStyleProject();
-        SecurityContext orig = ACL.impersonate(User.get("user").impersonate());

```

Comment:
(Note for me)
This test works even without this because security context is set to anonymous.
It fails if the security context is set to "admin" explicitly.